src/components/Atoms/Input/story.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Component from "./index";
+
+export default {
+  component: Component,
+  title: "Atoms/Input",
+  parameters: {
+    backgrounds: {
+      default: "dark",
+    },
+  },
+};
+
+const Template = (args) => <Component {...args} />;
+
+export const Default = Template.bind({});
+;
```

Comment:
```suggestion
 Default.args = {
   onEditComplete: (value) => console.log("edit completed: ", value),
 };
```
こんな感じで入力内容がコンソールに出力されるようにしてください！